test/parser_test.exs

```diff
@@ -122,6 +122,14 @@ defmodule ParserTest do
     assert opts[:content] == ~s(" text \#{content}\n")
   end
 
+  test "interpolation performance on long lines" do
+    slime = ~S(h2 Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do #{"eiusmod"})
+    {time, _} = :timer.tc(fn ->
+      Parser.parse_line(slime)
+    end)
+    assert time < 1000
```

Comment:
I don't understand how this is a useful test. It varies depending on the machine, the current workload, etc.